content/en/blog/falcoctl-gitlab-oci-artifacts-support.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "GitLab Container Registry now supports Falcoctl OCI Artifacts"
+linktitle: "GitLab Container Registry now supports Falcoctl OCI Artifacts"
+date: 2023-08-11
+author: Batuhan Apaydın
+slug: gitLab-supports-falcoctl-ociartifacts
+tags: ["Rules", "Falco Plugins", "Falcoctl", "Configuration Management"]
+---
+
+Today, we'd like to share with the Falco community the latest contribution we (w/[Emin Aktas](https://twitter.com/emminaktas)) made to [GitLab Container Registry](https://gitlab.com/gitlab-org/container-registry).
+
+We noticed that GitLab Container Registry didn't support falcoctl OCI Artifact [mediaTypes](https://github.com/falcosecurity/falcoctl/blob/7f1e8825a6f86010b9194577c56712dd0ef0442d/pkg/oci/constants.go#L20-L29) while we were pushing the Falco rules stored from GitHub container registry to GitLab container registry. We decided then to contribute to GitLab Container Registry by adding the support for falcoctl OCI Artifact mediaTypes.
+
+```shell
+Error: PUT https://registry.gitlab.com/v2/x/falcosecurity/rules/k8saudit-rules/manifests/1: MANIFEST_INVALID: manifest invalid; unknown media type: application/vnd.cncf.falco.rulesfile.config.v1+json
+Error: PUT https://registry.gitlab.com/v2/x/falcosecurity/plugins/k8saudit/manifests/sha256:b29c97a6590486f8b3b83644677e11d2f68e201a7035699189653d7f571d7e13: MANIFEST_INVALID: manifest invalid; unknown media type: application/vnd.cncf.falco.plugin.config.v1+json
+```
+
+You can learn more about our contribution [here](https://gitlab.com/gitlab-org/container-registry/-/merge_requests/1375). It will allow you to pull and push falcoctl OCI Artifacts from and to GitLab Container Registry once the feature is released, planned for GitLab **16.3**.
```

Comment:
```suggestion
You can learn more about our contribution [here](https://gitlab.com/gitlab-org/container-registry/-/merge_requests/1375). Once the feature is released, planned for GitLab **16.3**, it will allow you to pull and push Falcoctl OCI Artifacts from and to GitLab Container Registry.
```